### ISSUE TITLE
Audio prompts 

### ISSUE THREAD

**Issue by hentismith** [AndrewBelt/hack.chat#73]
Allow enable/disable of audio prompts on chat.

**Comment by ToastyStoemp** [comment]
Check the menu in the sidebar, should have 'Sound' that enables and disables audio

**Comment by AndrewBelt** [comment]
@ToastyStoemp That feature is only in a browser plugin.

**Comment by hentismith** [comment]
I'm using the https://hack.chat/ URL and sometimes miss conversations because there is no prompt that a response was int he "channel" hence requesting an audio prompt. 

Hope that clarifies things. If there is a chrome/firefox extention.plugin, please provide a link.

**Comment by M4GNV5** [comment]
https://github.com/ToastyStoemp/Hack.Chat-Enhancement-kit

**Comment by hentismith** [comment]
Thank you kindly :)

H

On Tue, Sep 22, 2015 at 1:34 PM, Jakob Löw notifications@github.com wrote:

> https://github.com/ToastyStoemp/Hack.Chat-Enhancement-kit
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/AndrewBelt/hack.chat/issues/73#issuecomment-142260715
> .

## 

##

**Comment by ToastyStoemp** [comment]
Yeah my bad, thought this was a thread on the issue page for the plugin

**Comment by masterpadam1122** [comment]
fuck you
